### PR TITLE
[MXNET-1291] solve pylint errors in examples with issue no.12205

### PR DESCRIPTION
**Pull request by cchung100m**
## Description ##
solve pylint errors in examples with issue #12205 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot add [pr-awaiting-review]

@vandanavk for review

File: example/autoencoder/model.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def extract_feature(sym, args, auxs, data_iter, N, xpu=mx.cpu()):
 
 
 class MXModel(object):
-    def __init__(self, xpu=mx.cpu(), *args, **kwargs):
+    def __init__(self, *args, xpu=mx.cpu(), **kwargs):
```
Comment (stu1130) [inline example/autoencoder/model.py]: this change looks weird to me

File: example/distributed_training/cifar10_dist.py [hunk 2]
```diff
@@ -124,38 +141,54 @@ def evaluate_accuracy(data_iterator, net):
...
 
     # Run the backward pass (calculate gradients) on all GPUs
     for l in losses:
         l.backward()
 
+
 # Train a batch using multiple GPUs
-def train_batch(batch, ctx, net, trainer):
+def train_batch(batch_list, context, network, gluon_trainer):
+    """ Training with multiple GPUs
```
Comment (stu1130) [inline example/distributed_training/cifar10_dist.py]:
```suggestion
    """ 
    Training with multiple GPUs
```

File: example/cnn_text_classification/data_helpers.py [hunk 3]
```diff
@@ -81,8 +83,8 @@ def pad_sentences(sentences, padding_word="</s>"):
     """
     sequence_length = max(len(x) for x in sentences)
     padded_sentences = []
-    for i in range(len(sentences)):
-        sentence = sentences[i]
+    for i, sentence in enumerate(sentences):
+        print(i, sentence)
```
[the remark this answers is not in the document]
Comment (cchung100m) [reply example/cnn_text_classification/data_helpers.py]: it seems like a testing code to me, I will remove it.

File: example/autoencoder/model.py (hunk 1 above)
Comment (cchung100m) [reply example/autoencoder/model.py]: I forget that why I change it, and it seems like the position of *args does not impact anything in the module, therefore I intend to remove the change if you agree.

**Comment by vandanavk** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Comment by vandanavk** [comment]
@sandeep-krishnamurthy for review/merge

**Comment by sandeep-krishnamurthy** [comment]
LGTM. @cchung100m - Can you please pull rebase, there is a flaky test that was fixed couple of days ago in the master branch.

**Comment by cchung100m** [comment]
Hi @sandeep-krishnamurthy

I complete the rebase and [re-test](https://gist.github.com/cchung100m/1cc785ef3fdf6cca90803ed84bc1a0b4) on this branch, I will appreciate if you can help to review/merge, thanks.

**Comment by cchung100m** [comment]
Hi @sandeep-krishnamurthy 

Could you please help to review or merge, thanks :)

**Comment by Roshrini** [comment]
Hi @cchung100m Can you resolve one conflicting file then we can merge this PR? Thanks!

**Comment by cchung100m** [comment]
Hi @Roshrini 

I reviewed the conflicting file and passed the CI, please help to merge this PR again, thanks. :)

**Comment by wkcn** [comment]
@cchung100m 
The PR has been merged. Thanks for your contribution!

**Pull request merged**